cfgov/unprocessed/css/search.less

```diff
@@ -22,7 +22,7 @@
   .content_sidebar {
     padding: 45px 15px 0;
     border: 1px solid @gray-40;
-    background-color: #f7f8f9;
+    background-color: @gray-5;
```

Comment:
I noticed this could also be a variable. It is not changing color, however, you can see gray 5 is f7f8f9 here https://cfpb.github.io/design-system/foundation/color